specification/communication/data-plane/Microsoft.CommunicationServicesAdministration/ga/2021-03-07/phonenumbers.json

```diff
@@ -0,0 +1,932 @@
+            "modelAsString": true
+          }
+        },
+        "capabilities": {
+          "$ref": "#/definitions/PhoneNumberCapabilities"
+        },
+        "assignmentType": {
+          "description": "The assignment type of the phone number.",
+          "enum": [
+            "Unknown",
+            "User",
+            "Application"
+          ],
+          "type": "string",
+          "x-ms-enum": {
+            "name": "PhoneNumberAssignmentType",
+            "modelAsString": true
+          }
+        },
+        "purchaseDate": {
+          "format": "date-time",
+          "description": "The date that the phone number is purchased.",
+          "type": "string"
+        },
+        "callbackUrl": {
+          "description": "The callback URL the phone number is assigned to.",
+          "type": "string"
+        },
+        "applicationId": {
+          "description": "The application id of the server application the phone number is assigned to.",
+          "type": "string"
+        },
+        "cost": {
+          "$ref": "#/definitions/PhoneNumberCost"
+        }
+      }
+    },
+    "CommunicationError": {
+      "description": "Represents the Communication Services error",
+      "required": [
+        "code",
+        "message"
+      ],
+      "type": "object",
+      "properties": {
+        "code": {
+          "description": "The error code.",
+          "type": "string"
+        },
+        "message": {
+          "description": "The error message.",
+          "type": "string"
+        },
+        "target": {
+          "description": "The error target.",
+          "type": "string",
+          "readOnly": true
+        },
+        "details": {
+          "description": "Further details about specific errors that led to this error.",
+          "type": "array",
+          "items": {
+            "$ref": "#/definitions/CommunicationError"
+          },
+          "readOnly": true
+        },
+        "innererror": {
+          "$ref": "#/definitions/CommunicationError"
+        }
+      }
+    },
+    "CommunicationErrorResponse": {
+      "description": "Represents the wrapper around the Communication Services error",
+      "required": [
+        "error"
+      ],
+      "type": "object",
+      "properties": {
+        "error": {
+          "$ref": "#/definitions/CommunicationError"
+        }
+      }
+    },
+    "PhoneNumberUpdateRequest": {
+      "description": "Represents a phone number update request",
+      "type": "object",
+      "properties": {
+        "callbackUrl": {
+          "description": "The webhook URL on the phone number configuration.",
+          "type": "string"
+        },
+        "applicationId": {
+          "description": "The application id of the application to which to configure",
+          "type": "string"
+        }
+      }
+    },
+    "AcquiredPhoneNumbers": {
+      "description": "A wrapper around a list of acquired phone numbers",
```

Comment:
"description": "The list of acquired phone numbers.",
 
 Terms like "wrapper" and such are technical and not really relevant, let's use simple language. From a developer's point of view, this is simply a list.

Reply:
Done.